docs/architecture/configuration-data-model.md

```diff
@@ -4,14 +4,14 @@
 
 This section enumerates the full set of features we expect to give to these models. Only the ones with the \(**MVP**\) tag are to be included in the MVP.
 
-### Persona: UI user
+### Persona: UI User
```

Comment:
not sure what the convention is. In the previous files you removed the capitalization